src/core/public/plugins/plugins_service.test.ts

```diff
@@ -97,9 +98,7 @@ beforeEach(() => {
   };
   mockStartContext = {
     ...omit(mockStartDeps, 'injectedMetadata'),
-    application: {
-      capabilities: mockStartDeps.application.capabilities,
-    },
+    application: expect.any(Object),
```

Comment:
what we want to test here?

Reply:
1. I'm not sure assertion works outside of expect. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/index.d.ts#L547-L563 this test uses it when creates a new object
2. In other places when we test interfaces it's excessive, because TS performs the same check